xmtp_mls/migrations/2024-05-15-145138_new_schema/up.sql

```diff
@@ -0,0 +1,104 @@
+-- Your SQL goes here
+CREATE TABLE openmls_key_store(
+    key_bytes BLOB PRIMARY KEY NOT NULL,
+    value_bytes BLOB NOT NULL
+);
+
+CREATE TABLE identity_inbox(
```

Comment:
The naming of `identity_inbox` was to avoid conflicts with legacy identity, can be `identity` now
```suggestion
CREATE TABLE identity(
```

Reply:
Will do